### ISSUE TITLE
AreaChart2 props not working

### ISSUE THREAD

**Issue by McFly78** [Abhinandan-Kushwaha/react-native-gifted-charts#892]
### Description

I am trying to  display a Line graph with 2 lines:
- The first line as an area graph
- the second line as only a line using the areaChart2 props set to false

As you can see both lines are displayed as an area graph.
<img width="261" alt="image" src="https://github.com/user-attachments/assets/358f2a19-7c05-461b-b750-443c964cd6c2">

As a workaround I temporarly modified the file /LineChart/index.js in the Gift-charts-core module.
line 1492 -> if ((areaChart ?? areaChart2) && data2.length > 0) {
replaced by: if (areaChart2 && data2.length > 0) {

Also is there a way to specify specific lineGradientId and areaGradientId for each line ?

Many thanks

### Steps to reproduce

```

import React from 'react';
import { View} from 'react-native';

import { LineChart} from "react-native-gifted-charts";

export default class MyChartTest extends React.PureComponent {
    constructor(props) {
      super(props);
    
    }
  
  render() {
    const lineData = [
      {value: 0},
      {value: 10},
      {value: 8},
      {value: 58},
      {value: 56},
      {value: 78},
      {value: 74},
      {value: 98},
    ];
    const lineData2 = [
      {value: 0},
      {value: 20},
      {value: 18},
      {value: 40},
      {value: 36},
      {value: 60},
      {value: 54},
      {value: 85},
    ];

    return (

      <View style={{borderWidth:1}}>
        <LineChart
          areaChart={true}
          areaChart2={false}
          curved
          data={lineData}
          data2={lineData2}
          height={250}
          showVerticalLines
          spacing={44}
          initialSpacing={0}
          color1="skyblue"
          color2="orange"
          textColor1="green"
          hideDataPoints
          dataPointsColor1="blue"
          dataPointsColor2="red"
          startFillColor1="skyblue"
          startFillColor2="orange"
          startOpacity={0.8}
          endOpacity={0.3}
          textShiftY={-2}
          textShiftX={-5}
          textFontSize={13}
        />
      </View>
    )
  }
}

```

### version of react-native-gifted-charts

1.4.46

### React Native version

0.74.4

### Platforms

iOS

### Workflow

React Native

**Comment by xuxiaoqian1** [comment]
AreaChart2、AreaChart3、AreaChart4、AreaChart5 props are not working。

**Comment by Abhinandan-Kushwaha** [comment]
Hi @McFly78 @xuxiaoqian1 Thanks for pointing out this issue.
It has been fixed from versions `1.4.48` onwards. Please use the latest version of the library.
